Update send-to-helix documentation (#2316)
dotnet/arcade: Documentation/AzureDevOps/SendingJobsToHelix.md

      EnableXUnitReporter: true # required for reporting out xUnit test results to Mission Control
      # WaitForWorkItemCompletion: true -- defaults to true
      Creator: arcade # specify an appropriate Creator here -- required for IsExternal true

>       # DisplayNamePrefix: 'Send job to Helix' -- the prefix of the task's display name on AzDO; the end will be " (Unix)" or " (Windows)"; defaults to 'Send job to Helix'

Reviewer: I do not understand how this value is used. Can you explain a bit more?

Author: It sets the text displayed here:

![image](https://user-images.githubusercontent.com/13228069/54794254-b9f58900-4c03-11e9-9128-d04ed0395c7b.png)

Reviewer: Gotcha. Trivial suggestion: how about something simplier? I do not see much value in listing the OS appends, plus it is a chance to be wrong if those ever change. 

```suggestion
      # DisplayNamePrefix: 'Send job to Helix' -- the Helix task's display name in AzDO. Defaults to 'Send job to Helix'
```

Author: Sensible; that definitely reads more clearly!
